src/sagemaker/tensorflow/README.rst

```diff
@@ -5,7 +5,26 @@ TensorFlow SageMaker Estimators allow you to run your own TensorFlow
 training algorithms on SageMaker Learner, and to host your own TensorFlow
 models on SageMaker Hosting.
 
-Supported versions of TensorFlow: ``1.4.1``, ``1.5.0``, ``1.6.0``, ``1.7.0``, ``1.8.0``, ``1.9.0``, ``1.10.0``, ``1.11.0``.
```

Comment:
sorry for my ambiguous comment last time. I think you can keep the list of links up here since presumably we'll eventually remove the warning box. in the warning box, I think you're good to keep just the latest legacy mode readme version